templates/jenkins-ssh-slave/config.yml

```diff
@@ -0,0 +1,5 @@
+name: Jenkins SSH Slave
+description: |
+  Add additional SSH slaves to enhance a Jenkins CI server.
+version: 0.01
```

Comment:
Build is failing due to error on `version` field. It should match to rancher-compose version.
```
-  version: 0.01
+  version: 0.0.1
```